### ISSUE TITLE
`PLURAL_ENCRYPTION_KEY_FILE` ignored by the console

### ISSUE THREAD

**Issue by maciejp-ro** [pluralsh/console#448]
https://github.com/pluralsh/console/blob/4aa48cec5aa564481a21636e76cce445d54bc34b/plural/helm/console/values.yaml.tpl#L86

This line always reads `~/.plural/key`, regardless of `PLURAL_ENCRYPTION_KEY_FILE` env variable

**Comment by michaeljguarino** [comment]
Good catch, but you also should be able to manually overwrite it and it'll take your update from then on if you want to avoid that default
